src/bucket/test/BucketListTests.cpp

```diff
@@ -140,9 +140,16 @@ TEST_CASE_VERSIONS("bucket list", "[bucket][bucketlist]")
                  !app->getClock().getIOContext().stopped() && i < 130; ++i)
             {
                 app->getClock().crank(false);
-                bl.addBatch(*app, i, getAppLedgerVersion(app), {},
-                            LedgerTestUtils::generateValidLedgerEntries(8),
-                            LedgerTestUtils::generateLedgerKeys(5));
+                bl.addBatch(
+                    *app, i, getAppLedgerVersion(app), {},
+                    LedgerTestUtils::generateValidUniqueLedgerEntries(8),
+                    LedgerTestUtils::generateValidLedgerEntryKeysWithExclusions(
+                        {
```

Comment:
This filter is necessary because there's only one possible key for CONFIG_SETTING, so a conflict is guaranteed if more than one of those entries exists.

Reply:
To add to that, the protocol check here is because `CONFIG_SETTING` only exists in vNext XDR, it's not intended to distinguish between the protocol versions in the future.

Reply:
oh I see so the problem is that we're pretty much guaranteed to generate conflicting entries for those entries. This makes sense.